packages/react/src/SelectPanel/SelectPanel.dev.stories.tsx

```diff
@@ -108,6 +126,7 @@ export const WithCss = () => {
           onSelectedChange={setSelected}
           onFilterChange={setFilter}
           className="testCustomClassnameMono"
+          message={isEmpty(selectedItemsSortedFirst, selected, filter) ? emptyMessage(filter) : undefined}
```

Comment:
```suggestion
          message={isEmpty(selectedItemsSortedFirst, selected, filter) ? <EmptyMessage filter={filter} /> : undefined}
```

If it's updated to a component above this could become this